samples/net/dsa/prj.conf

```diff
@@ -51,8 +51,8 @@ CONFIG_NET_CONFIG_INIT_TIMEOUT=10
 
 CONFIG_REQUIRES_FULL_LIBC=y
 
-CONFIG_NET_IF_MAX_IPV4_COUNT=4
-CONFIG_NET_IF_MAX_IPV6_COUNT=4
+CONFIG_NET_IF_MAX_IPV4_COUNT=10
+CONFIG_NET_IF_MAX_IPV6_COUNT=10
```

Comment:
Maybe these should be set by a board overlay for the RT1180? I doubt many boards are this capable as a switch, right?

Reply:
Different boards with different ethernet port number. This number here is not just for switch ports. There may be also other normal ethernet ports.
I'm okay to define in board overlay. But if max number 10 is accepted here, I think future boards no longer define in board overlay. 10 should be enough.

Reply:
Okay.